### PR TITLE
Feature/d3 asim 3274

### PR DESCRIPTION
**Pull request by faizan2590**

### CODE REVIEW

**Review by spyrostz — COMMENTED** [review]
2 comments:
- the interval events should also be renamed e.g. DisconnectIntervalAreaEvent and DisableIntervalAreaEvent
- the TransferFees namedtuple would be good to rename to GridFees

**Review by hannesdiedrich — COMMENTED** [review]
Sorry, pressed "Update Branch" by accident.
LGTM

File: src/d3a/models/market/__init__.py [hunk 1]
```diff
@@ -39,7 +39,7 @@
 
 log = getLogger(__name__)
 
-TransferFees = namedtuple("TransferFees", ('grid_fee_percentage', 'transfer_fee_const'))
+GridFee = namedtuple("GridFee", ('grid_fee_percentage', 'transfer_fee_const'))
```
Comment (hannesdiedrich) [inline src/d3a/models/market/__init__.py:42]: Sorry, if I missed the discussion, but shouldn't this also the opportunity to rename `transfer_fee_const` here too?
At least internally (we probably need a converter from d3a-web to d3a then)

**Review by Hassan754 — DISMISSED** [review]
LGTM

File: tests/test_market.py [hunk 2]
```diff
@@ -67,7 +67,7 @@ def __init__(self, bids=[], m_id=123, time_slot=now()):
         self.mcp_update_point = 20
         self.current_tick = 19
         # self.transfer_fee_ratio = transfer_fees.grid_fee_percentage / 100
-        # self.transfer_fee_const = transfer_fees.transfer_fee_const
+        # self.grid_fee_const = transfer_fees.grid_fee_const
```
Comment (hannesdiedrich) [inline tests/test_market.py:70]: I think you can remove this now.

**Review by hannesdiedrich — COMMENTED** [review]
Have you tried to start a simulation via d3a-web? 
from my memory, the renaming to `transfer_fee_const` was not that straight forward

**Comment by faizan2590** [comment]
> Have you tried to start a simulation via d3a-web?
> from my memory, the renaming to `transfer_fee_const` was not that straight forward

Yes, have tested to launch simulation via UI and targeting `d3a` at this branch

https://user-images.githubusercontent.com/33482427/108725674-e9674b80-7526-11eb-9a48-3c239a32cf02.mov

**Review by hannesdiedrich — APPROVED** [review]
LGTM

**Pull request merged**